lib/web/ui/server.go

```diff
@@ -162,7 +174,10 @@ func MakeKubeClusters(clusters []types.KubeCluster, userRoles services.RoleSet)
 // This function ignores any verification of the TTL associated with
 // each Role, and focuses only on listing all users and groups that the user may
 // have access to.
-func getAllowedKubeUsersAndGroupsForCluster(roles services.RoleSet, kube types.KubeCluster) (kubeUsers []string, kubeGroups []string) {
+func getAllowedKubeUsersAndGroupsForCluster(
+	roles services.RoleSet,
+	kube types.KubeCluster,
+) (kubeUsers []string, kubeGroups []string) {
```

Comment:
> The signature of a function or method declaration should remain on a single line to avoid [indentation confusion](https://google.github.io/styleguide/go/decisions#indentation-confusion).

See: https://google.github.io/styleguide/go/decisions#function-formatting

Reply:
That for some reason has been autoformatted that way. I just missed it.